Earthfile

```diff
@@ -660,18 +660,21 @@ netboot:
     ARG VERSION=$(cat VERSION)
     ARG ISO_NAME=${OS_ID}-${VARIANT}-${FLAVOR}-${TARGETARCH}-${MODEL}-${VERSION}
     ARG OSBUILDER_IMAGE
+    ARG RELEASE_URL=https://github.com/kairos-io/kairos/releases/download
+
+    ENV RELEASE_URL=$RELEASE_URL
```

Comment:
aren't `ARG`s automatically `ENV`s? I think there are several target where we don't do this kind of assignment and it works.

Reply:
mmmh, no idea, you might be right

Reply:
yepp, seems correct, let me drop them!